sessions/js-createelement/demo-end/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+		<title>HTML & CSS & JS</title>
```

Comment:
I would change the title :)
```suggestion
		<title>JS createElement demo-end</title>
```